### PR TITLE
My take on Dragon Hotend Toolhead for 2.2 and Voron Front Swivle LED Bar

### PR DESCRIPTION
**Pull request by orkusmg**
* [x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [x] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [x] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x] Submitted STL files are oriented and scaled properly for printing.
  * [x] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?

Toolhead mod for Dragon Hotend

#### How was it tested? 

I'm running it in my Voron

### CODE REVIEW

File: printer_mods/OrkusMG/DragonToolhead/ReadMe.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Triangle Lab Dragon Hotend Toolhead
+
+This toolhead mount design is based of Voron 2.2 Toolhead files.
+
+It is one piece print, designed to work with everything you have included in you Dragon Hotend delivery package (screws and bowden collete) - no additional extra parts required.
+
+For printing I advice using bit of support for the face marked in blue visible on the Back picture below.
+
+
+![Front](https://github.com/orkusmg/VoronUsers/blob/master/printer_mods/OrkusMG/DragonToolhead/Front.jpg)
```
Comment (FHeilmann) [inline printer_mods/OrkusMG/DragonToolhead/ReadMe.md]: Use relative links (./Front.jpg)

File: printer_mods/OrkusMG/DragonToolhead/ReadMe.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+![Bottom](https://github.com/orkusmg/VoronUsers/blob/master/printer_mods/OrkusMG/DragonToolhead/Bottom.jpg)
```
Comment (FHeilmann) [inline printer_mods/OrkusMG/DragonToolhead/ReadMe.md]: Use relative links (./Bottom.jpg)

File: printer_mods/OrkusMG/DragonToolhead/ReadMe.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+![Top](https://github.com/orkusmg/VoronUsers/blob/master/printer_mods/OrkusMG/DragonToolhead/Top.jpg)
+
+![Back, or bottom if you take print orientation - where support is required](https://github.com/orkusmg/VoronUsers/blob/master/printer_mods/OrkusMG/DragonToolhead/Back.jpg)
```
Comment (FHeilmann) [inline printer_mods/OrkusMG/DragonToolhead/ReadMe.md]: Use relative links (./Back.jpg)

File: printer_mods/OrkusMG/VoronFrontSwivelLED/ReadMe.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+No supports required during print.
+
+You will need:
+2x M3x6
+2x M3x8
+4x M3 nuts for extrusion slot mounting.
+
+
+
+![1](https://github.com/orkusmg/VoronUsers/blob/master/printer_mods/OrkusMG/VoronFrontSwivelLED/1.jpeg)
```
Comment (FHeilmann) [inline printer_mods/OrkusMG/VoronFrontSwivelLED/ReadMe.md]: Use relative links (./1.jpeg)

File: printer_mods/OrkusMG/VoronFrontSwivelLED/ReadMe.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+![2](https://github.com/orkusmg/VoronUsers/blob/master/printer_mods/OrkusMG/VoronFrontSwivelLED/2.jpeg)
```
Comment (FHeilmann) [inline printer_mods/OrkusMG/VoronFrontSwivelLED/ReadMe.md]: Use relative links (./2.jpeg)

File: printer_mods/OrkusMG/VoronFrontSwivelLED/ReadMe.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+
+![3](https://github.com/orkusmg/VoronUsers/blob/master/printer_mods/OrkusMG/VoronFrontSwivelLED/3.jpeg)
```
Comment (FHeilmann) [inline printer_mods/OrkusMG/VoronFrontSwivelLED/ReadMe.md]: Use relative links (./3.jpeg)

File: printer_mods/OrkusMG/VoronFrontSwivelLED/ReadMe.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+
+![4](https://github.com/orkusmg/VoronUsers/blob/master/printer_mods/OrkusMG/VoronFrontSwivelLED/4.jpeg)
```
Comment (FHeilmann) [inline printer_mods/OrkusMG/VoronFrontSwivelLED/ReadMe.md]: Use relative links (./4.jpeg)

File: printer_mods/OrkusMG/DragonToolhead/ReadMe.md [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
+It is one piece print, designed to work with everything you have included in you Dragon Hotend delivery package (screws and bowden collete) - no additional extra parts required.
+
+For printing I advice using bit of support for the face marked in blue visible on the Back picture below.
+
+
+![Front](https://github.com/orkusmg/VoronUsers/blob/master/printer_mods/OrkusMG/DragonToolhead/Front.jpg)
+
+![Bottom](https://github.com/orkusmg/VoronUsers/blob/master/printer_mods/OrkusMG/DragonToolhead/Bottom.jpg)
+
+![Top](https://github.com/orkusmg/VoronUsers/blob/master/printer_mods/OrkusMG/DragonToolhead/Top.jpg)
```
Comment (FHeilmann) [inline printer_mods/OrkusMG/DragonToolhead/ReadMe.md]: Use relative links (./Top.jpg)

**Comment by orkusmg** [comment]
All changes done, links are now relative and model contains break-away support block, so no supports from slicer are required to print correctly. 

PS: plase also check my VoronFrontSwivelLED folder I submited as PR.

**Pull request merged**